### ISSUE TITLE
💡 Coursemology Week 3 Question 10: Ex10

### ISSUE THREAD

**Issue by choongzhanhong** [nus-cs2113-AY2223S2/forum#9]
Hi folks, this was originally supposed to be a help request for [Ex10](https://nus-cs2113-ay2223s2.github.io/website/schedule/week3/topics.html#key-exercise-find-total-expenditure), but I found the solution while writing this, so I'm sharing with the rest of you.

Note that the desired output is this:
`Your expenses while overseas?Expenses in overseas currency:[$4.50, $3.00, $5.00]
Total in local currency: $21.25`

The skeleton code does not use `println` but instead uses `print` (Does not automatically go to a newline). Also note there is no space between `"Expenses in overseas currency:"` and your array. These have to match the output on Coursemology to be considered correct.

Honestly, the desired output is not very nice because of inconsistent spacing, but oh well. Coursemology asks, I give.

**Comment by lihka1202** [comment]
Yep, aside from the spacing and `println` vs `print`, the other important thing is to format the final line properly. Just using `.toString` would yield `0.0` if the sum is 0, however coursemology requires the output to be `0.00`.

**Comment by okkhoy** [comment]
Errm.. thanks for pointing it out. Let me get some of these fixed.
`print` vs `println` was a conscious choice due to some Coursemology limitations.
